### DISCUSSION TITLE
win端打开后，复制文字到软件里面，点击“翻译”等按钮没反应。key都填了

### DISCUSSION THREAD

**Discussion by alanjustalan** [nextai-translator/nextai-translator#201] [Q&A]
https://user-images.githubusercontent.com/42884681/224474913-8dfbc484-d444-4cba-9a20-7bb9171b4980.mp4

**Comment by Bm7651** [comment]
我也遇到了同样的问题，请问你解决了吗

**Reply by tcorbusier** [reply]
我也遇到一样的问题 win11系统

**Comment by yetone** [comment]
抱歉，按一下回车键即可，这是交互上的失误，我会加上提醒或者按钮

**Reply by Bm7651** [reply]
您好，我的问题是不管在 win 端还是扩展端，在正常填写 api，且具备正常访问 chatgpt 官网的条件下，总是出现这个“TypeError:Failed to fetch”api以明确验证可以正常使用
